apps/nextra/pages/en/build/smart-contracts/linter.mdx

```diff
@@ -12,6 +12,15 @@ This tool is currently in beta, so please try it out and submit [bugs and feedba
 
 
 ## Lint Checks
+### `almost_swapped`
+Checks for expression patterns that look like a failed swap attempt and notifies the user. These patterns are likely erroneous code. This currently only detects simple access patterns including assignments to a variable or a field of a struct. Examples include:
+- `a = b; b = a;` which can be correctly swapped with `(a, b) = (b, a);`
+- `a.x = b.x; b.x = a.x;` which can be correctly swapped with `(a.x, b.x) = (b.x, a.x);`
+
+Note that this does not detect patterns involving non-builtin functions, including vector operations, nor does this detect patterns that deal with global storage. For example, the following are out of scope of this lint check:
+- `v[0] = v[1]; v[1] = v[0];`
+- `*borrow_global_mut<Coin>(addr1) = *borrow_global_mut<Coin>(addr2); *borrow_global_mut<Coin>(addr2) = *borrow_global_mut<Coin>(addr1);` 
```

Comment:
The modern Move 2 syntax for this would be `Coin[addr1] = Coin[addr2]; ...`

Reply:
Didn't realize this. Thank you!